content/blog/2023/2023-08-30-fairness-equal-odds-postprocessing.md

```diff
@@ -0,0 +1,57 @@
++++
+author = "Žan Mervič"
+date = "2023-08-30"
+draft = false
+title = "Orange Fairness - Equal Odds Postprocessing"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-30-fairness-equal-odds-postprocessing.png"
+frontPageImage = "/blog_img/2023/2023-08-30-fairness-equal-odds-postprocessing.png"
+blog = ["fairness", "equal odds postprocessing"]
+shortExcerpt = "In this blog, we delve into the Equal Odds Postprocessing widget, a tool designed to enhance fairness in machine learning models. We break down how the algorithm works by modifying predictions to meet Equalized Odds criteria. Using a real-world example with the German credit dataset, we demonstrate its efficacy in improving fairness metrics while marginally affecting accuracy."
+longExcerpt = "Explore the Equal Odds Postprocessing widget in Orange, designed to fine-tune your model's fairness. We explain how the algorithm operates and showcase its effectiveness with an example using the German credit dataset."
++++
+
+
+In the [previous blog post]({{< ref "2023-08-28-fairness-adversarial-debiasing.md" >}}), we discussed the Adversarial Debiasing model, a bias-aware model. This blog post will discuss the Equal Odds Postprocessing widget, a bias-aware post-processor, which can be used with any model to mitigate bias in its predictions.
+
+### Equal Odds Postprocessing:
+
+The [Equal Odds Postprocessing](https://arxiv.org/abs/1610.02413) widget is a post-processing type of fairness mitigation algorithm designed to ensure fairness in supervised learning. It modifies the predictions of any given classifier to meet certain fairness criteria, specifically focusing on "Equalized Odds" or more relaxed criteria like Equal Opportunity. Because it is a post-processing algorithm, it does not require any support from the model (like reweighing, which requires the model to support weights) or changes to the model (like adversarial debiasing, which requires the model to be a neural network). This makes it a versatile algorithm that can be used with any model.
```

Comment:
designed to ensure fairness in supervised learning -> "for supervised learning" (the reader knows from the rest of the sentence that this is about fairness.

any support from the model - strange and unclear phrasing, I'd do this:
"it does not require any support from the model (like reweighing, which requires the model to support weights) or changes to the model (like adversarial debiasing, which requires the model to be a neural network). This makes it a versatile algorithm that can be used with any model." -> "it works with general supervised models."